pkg/aws/actuator/actuator.go

```diff
@@ -227,9 +227,23 @@ func (a *AWSActuator) needsUpdate(ctx context.Context, cr *minterv1.CredentialsR
 
 	} else {
 		// for passthrough creds, just see if we have the permissions requested in the credentialsrequest
```

Comment:
In the original PR, this line was removed. Any reason this PR isn't a cherry pick of #242

Reply:
Yes, looks like cherry pick is a better option here. I thought some changes won't go in 4.6, but I was wrong. I will update the PR. Thanks!

Reply:
Removed this line.

Reply:
lets go ahead with this PR as automated cherry pick of #242 [resulted in conflicts](https://github.com/openshift/cloud-credential-operator/pull/242#issuecomment-749090096). Removed the above line.